src/Summoner/CLI.hs

```diff
@@ -128,14 +169,31 @@ summonerVersion = toString $ intercalate "\n" $ [sVersion, sHash, sDate] ++ [sDi
 -- All possible commands.
 summonerP :: Parser Command
 summonerP = subparser
-    ( command "new"
-      (info (helper <*> newP) $ progDesc "Create a new Haskell project")
-    )
+    $ command "new" (info (helper <*> newP) $ progDesc "Create a new Haskell project")
+   <> command "show" (info (helper <*> showP) $ progDesc "Show available licenses or ghc versions")
 
 ----------------------------------------------------------------------------
 -- New command parsers
 ----------------------------------------------------------------------------
 
+-- | Parses options of the @show@ command.
+showP :: Parser Command
+showP = ShowInfo <$> (ghcP <|> ShowLicense <$> licenseP)
```

Comment:
I think we should be consistent in the choice of the command/flag.
Either both of the `ghc` and `licence` should be commands, or both of them flags.
(I guess the command is preferable)